packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -80,6 +80,7 @@ export enum EmailEventStatusEnum {
   BLOCKED = 'blocked',
   SPAM = 'spam',
   UNSUBSCRIBED = 'unsubscribed',
+  INVALID = 'invalid',
```

Comment:
this should be on the `NetCoreStatusEnum` right?

Reply:
Yes, and I think we can remove it from `EmailEventStatusEnum` since we're mapping the `NetCoreStatusEnum.INVALID` to the BOUNCED status?

Reply:
yes